packages/rke2-calico/generated-changes/patch/values.yaml.patch

```diff
@@ -37,7 +37,7 @@
  calicoctl:
 -  image: docker.io/calico/ctl
 +  image: rancher/mirrored-calico-ctl
-   tag: v3.23.1
+   tag: v3.23.3
```

Comment:
You should also move the tigera operator version to `v1.27.12`, which seems to be the latest one to install v3.23.3 ==> [v1.27.12](https://github.com/tigera/operator/releases/tag/v1.27.12)

Reply:
Thanks for your comment. Fixed.